apps/mapideas/templates/meinberlin_mapideas/mapidea_detail.html

```diff
@@ -12,6 +12,6 @@
 
 {% block additional_content %}
     {% if object.point %}
-        {% map_display_point object.point object.module.settings_instance.polygon %}
+        {% map_display_point object.point object.module.area_settings.polygon %}
```

Comment:
This shouldn't work because the realname of that reverse relation is `areasettings_settings`, I updated my other comment a few minutes later. Sorry.